geometry/list-transformations-by-suitability/README.md

```diff
@@ -27,6 +27,22 @@ If the selected transformation is not usable (has missing grid files) then an er
 * GeometryEngine
 * TransformationCatalog
 
+## Additional information
+
+Some transformations aren't available until transformation data is provided.
+
+This sample uses a `GeographicTransformation`, which extends the `DatumTransformation` class. As of 100.9, ArcGIS Runtime also includes a `HorizontalVerticalTransformation`, which also extends `DatumTransformation`. The `HorizontalVerticalTrasnformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
```

Comment:
Should this be `Z-aware`? Not sure when you capitalise.

```suggestion
This sample uses a `GeographicTransformation`, which extends the `DatumTransformation` class. As of 100.9, ArcGIS Runtime also includes a `HorizontalVerticalTransformation`, which also extends `DatumTransformation`. The `HorizontalVerticalTrasnformation` class is used to transform coordinates of Z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
```

Reply:
Not 100%, seems to be z-aware when I did a search on documentation, but refer to a Z when there's not a hyphen. I'll leave as is based on the documentation check.